app/views/custom_styles/_primer_color_mapping.erb

```diff
@@ -86,5 +86,7 @@
       --main-menu-bg-color: var(--overlay-bgColor);
       --type-form-conf-attribute--background: var(--overlay-bgColor);
       --select-arrow-bg-color-url: url("data:image/svg+xml;utf8,<svg fill='white' height='24' viewBox='0 0 24 24' width='24' xmlns='http://www.w3.org/2000/svg'><path d='M7 10l5 5 5-5z'/><path d='M0 0h24v24H0z' fill='none'/></svg>");
+      --button--primary-background-disabled-color: var(--primary-button-color--major1);
```

Comment:
You probably also have to override the `--button-primary-borderColor-disabled`. Otherwise it looks a bit weird.
<img width="190" alt="Bildschirmfoto 2024-07-12 um 11 10 49" src="https://github.com/user-attachments/assets/2be3dece-4625-4fe9-96fe-855744c110ee">

This is how it should like [in Primer](https://primer-lookbook.github.com/view-components/lookbook/inspect/primer/beta/button/playground?_display=%257B%2522theme%2522%253A%2522dark%2522%257D&scheme=primary&inactive=false&disabled=true):

<img width="102" alt="Bildschirmfoto 2024-07-12 um 11 11 46" src="https://github.com/user-attachments/assets/a3537eaf-ba26-4ed0-b9db-7d7fa07a748f">